threads.go

```diff
@@ -350,6 +344,12 @@ func (t *threads) AddRecord(
 		return
 	}
 
+	// Push out the new record
```

Comment:
So prefer succeed sending to bus if pushRecord fails?

Reply:
At this point, the record has been committed... network failure is a different concern

Reply:
:+1: